lib/ansible/plugins/connection/network_cli.py

```diff
@@ -6,42 +6,143 @@
 __metaclass__ = type
 
 DOCUMENTATION = """
-    author: Ansible Networking Team
-    connection: network_cli
-    short_description: Use network_cli to run command on network appliances
+---
+author: Ansible Networking Team
+connection: network_cli
+short_description: Use network_cli to run command on network appliances
+description:
+  - This plugin actually forces use of 'local' execution but uses paramiko to
+    establish a remote ssh shell on the network device.  This connection
+    plugin will load two additional plugins to manage the session.  It will
+    load a terminal plugin to setup terminal parameters and a cliconf plugin
+    to handle serializing requests to the remote device.  Both plugins are
+    loaded based on the network_os configuration value (see options below).  If
+    the network device needs privilege escalation (for instance 'enable' mode),
+    set the become directive in the playbook for the play or task (or configure
+    it in inventory)
+version_added: "2.3"
```

Comment:
`version_added: "2.5"`?

Reply:
No, it's been here since 2.3, just not usable as a connection on its own.